source/localizable/_code-of-conduct.it.md

```diff
@@ -0,0 +1,67 @@
+
+
+Cittadinanza Open [Source/Culture/Tech]
+--------------------------------------
+
+Uno degli scopi addizionali di questo Codice di Condotta è di favorire una cittadinanza open [source/culture/tech] incoraggiando i partecipanti a riconoscere e rafforzare le relazioni tra le nostre azioni e come esse influenzano la nostra community.
+
+Le communities sono lo specchio delle società nelle quali esse esistono ed agire in maniera positiva è essenziale per contrastare tutte le forme di inequalità e abusi di potere che esistono nella società in cui viviamo.
+
+
+Se vieni a conoscenza di qualcuno/a il/la cui esemplarmente si assicura che la nostra community sia accogliente, amichevole, e che attivamente incoraggia tutti i partecipanti nel contribuire al massimo, faccelo sapere!
+
+
+Il comportamento che ci aspettiamo
+-----------------
+
+* Partecipa in maniera autentica e attiva. Facendo ciò, stai contribuendo nell' avere una community sana e longeva.
+* Sii considerato e rispettoso riguardo a cosa dici e a come agisci.
+* Cerca di collaborare prima di entrare in conflitto.
+* Astieniti da commenti, discussioni o comportamenti avvilenti, discriminatori o molesti.
+* Sii conscio dell' ambiente e dei suoi partecipanti. Se noti una situazione pericolosa, qualcuno che non si sente a suo agio, o una violazione di questo Codice di Condotta, avvisa immediatamente i leaders della community, anche se la situazione non sembra poter avere conseguenze particolarmente dannose.
+
+
+Comportamento inaccettabile
+---------------------
+
+I comportamenti che riteniamo inaccettabili includono: intimidatori, molesti, abusanti, discriminatori, linguaggi o azioni dispregiativi e avvilenti perpetuati da qualsiasi partecipante nella nostra community online, in tutti gli eventi collegati e nelle conversazioni tra singoli pertinenti agli interessi della community. Alcuni eventi della Community potrebbero aver luogo in spazi condivisi con un pubblico esterno alla community stessa; per favore, sii rispettoso/a con tutti/e i/le promotori/e dei suddetti spazi.
+
+Comportamenti molesti includono: commenti scritti o verbali dannosi o pregiudizievoli in relazione al sesso, orientamento sessuale, razza, religione, disabilità fisiche e/o mentali; uso inappropriato di immagini di nudità e/o immagini a sfondo sessuale utilizzate in spazi pubblici (comprese le slides per le presentazioni); intimidazioni, il seguire fisicamente o stalking; molestie registrate sotto forma di fotografie e video; interruzione sostenuta di presentazioni o altri eventi; contatto fisico inappropriato, e sgradita attenzione sessuale.
+
+
+Conseguenze per i comportamenti inaccettabili
+-------------------------------------
+
+Qualsiasi comportamento ritenuto inaccettabile perpetuato da qualsiasi membro della community, compresi Sponsors e coloro nella posizione di potere decisionale, non sarà tollerato.
```

Comment:
1. perpetrato
2. "e coloro che si trovano in una posizione di potere decisionale"